### ISSUE TITLE
About conda install -c conda-forge gdal nco cdo wget

### ISSUE THREAD

**Issue by nabilabdessattar** [bennyistanto/spi#1]
Dear Benny, please would you help me to solve this issue, once I tried to install gdal nco and cdo in gis environment, I got an error in solving environment process as failed with initial frozen solve. I use windows 11 machine and followed all steps until this one doesn't work; Thank you in advance

**Comment by bennyistanto** [comment]
You can create 2 environment, climate_indices which consist of nco netcdf4 and for calculating the spi, and another environment called "gis" or whatever the name which consist of gdal and cdo only use for translating the result into geotiff.

**Comment by nabilabdessattar** [comment]
Thank you Benny for your quick reply, all the best

**Comment by nabilabdessattar** [comment]
Dear Benny, thank you for your support, it works. Please, I would like to verify that during creating folders where downloads folder is there for chirps data, Do I have to create them in windows explorer and later access them from ubuntu terminal using change directory command cd C:\Temp\CHIRPS\SPI\Java\Downloads\CHIRPS\GeoTIFF\ 
or we need to create the different folders you mentioned in working directory using command in ubunto in my home with gis envi activated? thank you benny

**Comment by bennyistanto** [comment]
You can use whatever folder structures you have (my folder structure is just an example). Creating from both windows explorer and via ubuntu terminal is same.

`cd C:\Temp\CHIRPS\SPI\Java\Downloads\CHIRPS\GeoTIFF\` will nor work in Ubuntu.

In Ubuntu Terminal, navigate to the home user's directory, then go to the C drive

```
cd ~
cd ../../../mnt/c/Temp/CHIRPS/SPI/Java/Download/CHIRPS/GeoTIFF
```

**Comment by nabilabdessattar** [comment]
Thank you Benny,
regards

**Comment by nabilabdessattar** [comment]
Sorry Benny, again I have an issue with downloading files, I got this message
![image](https://user-images.githubusercontent.com/123644825/220173784-b320189b-8791-4da0-88a8-00898f6c4ba0.png)

**Comment by bennyistanto** [comment]
If using script doesn't worked for you, you can manually download it from here https://data.chc.ucsb.edu/products/CHIRPS-2.0/global_monthly/tifs/ or you can try using ftp client in Windows like FileZilla or other.

**Comment by nabilabdessattar** [comment]
Ok Benny, thank you for your support,
regards

**Comment by nabilabdessattar** [comment]
Hi Benny, thank you for your support and steps are going well, however, I got an error in converting Geotiff files to nc, I run python code but I get an error as following
![image](https://user-images.githubusercontent.com/123644825/220548777-dbac2a72-88fe-47f5-80e3-6f89e6adf937.png)

**Comment by nabilabdessattar** [comment]
regarding the previous message, I alreadu installed python through conda

**Comment by bennyistanto** [comment]
You don't have `gdal` in your environment. Install it using `conda install -c conda-forge gdal`

**Comment by nabilabdessattar** [comment]
Thank you Benny, but still I got this message

![image](https://user-images.githubusercontent.com/123644825/220555245-e6ef65a5-c0cf-4d7d-8c79-50d942dc3265.png)

**Comment by bennyistanto** [comment]
Change `import gdal` to `from osgeo import gdal` in your script. Hope it will work.

**Comment by nabilabdessattar** [comment]
Thank you Benny, it seems Ok, but I got another error message about netCDF module. I installed again but it is not working
![image](https://user-images.githubusercontent.com/123644825/220570292-8cf6b488-5acb-4574-8b69-d339ee87df3a.png)

**Comment by bennyistanto** [comment]
I assume you have installed with this `conda install -c conda-forge netcdf4`

If you are still getting an error, then I can't help. Sorry.

**Comment by nabilabdessattar** [comment]
yes I did Benny I installed with with conda install -c conda-forge netcdf4
Thank you again Benny
